src/arch/host/include/arch/debug/panic.h

```diff
@@ -5,7 +5,7 @@
  * Author: Tomasz Lauda <tomasz.lauda@linux.intel.com>
  */
 
-#ifdef __XTOS_RTOS_PANIC_H__
+#ifdef __POSIX_RTOS_PANIC_H__
```

Comment:
Hmm, continuing with previous, @lrgirdwo @ranj063 , I don't see a point in arch layer if we go for indepdent "rtos" implementation ("XTOS", "Zephyr" and "posix").

Reply:
We are stuck a bit today whilst xtos is still being used, i.e. difficult to extract until more xtos related code can be removed (and this depends on progress with Zephyr integration for all platforms).